tarball/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/Scalingo/go-utils/tarball
+
+go 1.13
+
+require (
+	github.com/Scalingo/go-utils v6.4.0+incompatible
+	github.com/Scalingo/go-utils/fs v0.0.0-20200427101022-73f3c803d841
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.5.0 // indirect
+	github.com/spf13/afero v1.2.2
+	github.com/stretchr/testify v1.2.2
```

Comment:
Update to the latest 1.5.1? We will forget to update it in the future so at least use the latest version at the time of release :P

Reply:
Done